web/packages/core/src/load-options.ts

```diff
@@ -214,6 +214,13 @@ export interface BaseLoadOptions {
      */
     contextMenu?: boolean;
 
+    /**
+     * Whether or not to show a preloader before the SWF has loaded with Ruffle.
+     *
+     * @default true
+     */
+    hasPreloader?: boolean;
```

Comment:
I would even shorten this into just `preloader`. It will also open up the possibility for non-boolean values in the future.

Reply:
I can do so for the configuration option, but I'll leave the private ruffle-player variable as hasPreloader unless you have a different naming suggestion for the HTML Element because `ruffle-player.preloader` is currently the HTML element itself.

Reply:
Looks good as it now!